pages/bellapps.md

```diff
@@ -34,22 +34,24 @@ Fill out the information, although as we are just doing this as a test, accuracy
 In case you forgot to take the screenshot of your configurations, go to http://127.0.0.1:5985/apps/_design/bell/MyApp/index.html, then click on manager, click on the configurations tab, take a screenshot of the page, and submit it to the Gitter chat.
 
 ## Different Kinds of Updates to Your Community
-There are three other important kinds of updates that you receive on the community side: updates, publications, and surveys. 
+There are three other important kinds of updates that you receive on the community side: updates, publications, and surveys.
 
 As you can see from the image below, there is an update ready to be downloaded. Usually, next to the update, you should also see  publication(s) ready to be downloaded.
 
 ![Update from the nation](uploads/images/update_publication.jpg "Dashboard in your localhost")
 
 First, click the "Update Available" button and it will reload your homepage with a successful update message. An update refers to a new software update which improves the software. Next, click on "Publications", under the Manager page, and sync the publications. Publications add new resources or courses to your library. Last, repeat the process of sending an activities sync to the nation.
 
+**NOTE**: If `Update Available` or `Publications`, is not visible, continue on with tutorial. They will only display if necessary!
```

Comment:
Is the `,` after `Publications` necessary? Would it be more friendly to use `.` instead of `!` at the end of the sentence?